docs/docs-app/components/app.js

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
+import React, {Component} from 'react';
 
 import Header from './header';
 
 /* eslint-disable react/display-name*/
-export default ({children}) => (
-  <div className="app">
-    <Header />
-    <div className="app-container">
-      {children}
-    </div>
-  </div>
-);
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      opacity: 1,
+      isMenuOpen: false
+    };
+    this.toggleMenu = this.toggleMenu.bind(this);
+  }
+  toggleMenu() {
+    this.setState({isMenuOpen: !this.state.isMenuOpen});
+  }
+  render() {
+    return (<div className="app">
+      <Header
+        isMenuOpen={this.state.isMenuOpen}
```

Comment:
in a previous PR you were asserting that we tried to stray away from destructuring, however this hasn't been the case! destructuring is good! looks good feels good.